### PR TITLE
v4.4.0.0 release

### PR DESCRIPTION
**Pull request by borditamas**
TODO: change `arrowhead-application-library-java-spring` staging repository to central repository before merge (but only after release)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -9,32 +9,39 @@
...
+	    <module>application-skeleton-publisher</module>
   	</modules>
-		
+	
+	<repositories>
+		<repository>
+			<id>ossrh</id>
+			<url>https://s01.oss.sonatype.org/content/repositories/aiaitia-1017</url>
+		</repository>
+	</repositories>
```
Comment (rbocsi) [inline pom.xml]: is this necessary after the application-library will be available in maven central?

**Review by rbocsi — APPROVED** [review]
I have only one comment (see inline)

Edit: you mentioned that in your initial comment, so I closed that comment. However, I found a problem.

File: pom.xml (hunk 1 above)
Comment (rbocsi) [reply pom.xml]: ok, I see your comment now.

File: README.md [hunk 2]
```diff
@@ -10,7 +10,6 @@ Fork this repo and extend the skeletons with your own application code. ([check
 The project has the following dependencies:
```
Comment (rbocsi) [inline README.md:10]: The README still contains the word 'client' multiple times, including the title.

**Review by rbocsi — CHANGES_REQUESTED** [review]
found a problem

File: README.md (hunk 2 above)
Comment (borditamas) [reply README.md:10]: resolved

**Review by rbocsi — APPROVED** [review]

**Pull request merged**